respotter.py

```diff
@@ -26,6 +27,18 @@ def __init__(self,
         self.hostname = hostname
         self.timeout = timeout
         self.verbosity = verbosity
+
+    def get_arguments(self):
+        parser = OptionParser()
+        parser.add_option("-d", "--delay", dest="delay", help="Delay between scans in seconds", default=30)
+        parser.add_option("-t", "--timeout", dest="timeout", help="Timeout for each scan in seconds", default=3)
+        parser.add_option("-v", "--verbosity", dest="verbosity", help="Verbosity level (0-3)", default=0)
+        parser.add_option("-n", "--hostname", dest="hostname", help="Hostname to scan for", default="Loremipsumdolorsitamet")
+        (options, args) = parser.parse_args()
+        self.delay = int(options.delay)
+        self.timeout = int(options.timeout)
+        self.verbosity = int(options.verbosity)
+        self.hostname = options.hostname
```

Comment:
This works for our use case, but it might make more sense to move the argument parsing to the "main" block below (line 81-82) so that it only parses command line arguments when the file is executed as a script. Then instead of assigning them to `self.<attribute>`, you would just pass them in when you create the `Respotter()` object.

A situation where this might make a difference is if we use the `Respotter` class as an import in another Python program. The command line arguments might not be the same in that program, so we wouldn't need to have the argument parsing functionality as part of the object class.

Other than the rearrangement, the code looks good! Nice work.